migrations/3_batch_auction.js

```diff
@@ -1,16 +1,13 @@
 /*eslint no-undef: "off"*/
 
 const BiMap = artifacts.require("IdToAddressBiMap.sol")
-const IterableAppendOnlySet = artifacts.require("IterableAppendOnlySet.sol")
```

Comment:
Turns out this is actually breaking the e2e tests when syncing with dex-services.

Reply:
I had a similar issue with the contract code base yesterday on my local (running gas estimations). I believe I managed to recover from this with a good-ol' `npm install`. Could this be the issue?